Dockerfile

```diff
@@ -529,9 +535,6 @@ RUN --mount=type=cache,sharing=locked,id=moby-dev-aptlib,target=/var/lib/apt \
             net-tools \
             patch \
             pigz \
-            python3-pip \
-            python3-setuptools \
-            python3-wheel \
```

Comment:
looks like yamllint was happy to install without these, but maybe they were needed for cross-compile, so let's see if CI is happy.

Reply:
I think this was needed for `pip3 install`, to pin to a specific version of yamlint